torch/_dynamo/variables/tensor.py

```diff
@@ -69,6 +69,7 @@
     "<=": operator.le,
     "==": operator.eq,
     "!=": operator.ne,
+    "is": operator.is_,
```

Comment:
Can you also add support for: operator.is_not(a, b)[¶](https://docs.python.org/3/library/operator.html#operator.is_not)